src/python/library/tritonclient/grpc/__init__.py

```diff
@@ -637,7 +646,11 @@ def load_model(self, model_name, headers=None, config=None):
             if config is not None:
                 request.parameters["config"].string_param = config
             if self._verbose:
-                print("load_model, metadata {}\n{}".format(metadata, request))
+                # Don't print file content which can be large
+                print("load_model, metadata {}\noverride files omitted:\n{}".
+                      format(metadata, request))
+            for path, content in encoded_files.items():
```

Comment:
> If specified, 'config' must be provided to be the model configuration of the override model directory.

Do we need to check that config is provided (non-empty) before doing something with encoded_files? Same in http client too

Reply:
I didn't add the checking here, but simply rely on the server to check and to return proper error. We could add the check in client as well but that will intercept the server-side checking and make it harder to test server behavior.